integration-tests/chopsticks/src/network/spiritnet.ts

```diff
@@ -0,0 +1,60 @@
+import { setupContext, SetupOption } from '@acala-network/chopsticks-testing'
+import type { Config } from './types.js'
+import * as HydraDxConfig from './hydroDx.js'
+
+export const options: SetupOption = {
+	endpoint: 'wss://kilt-rpc.dwellir.com',
+	db: './db/spiritnet.db.sqlite',
+	port: 8002,
+	wasmOverride: '../../target/debug/wbuild/spiritnet-runtime/spiritnet_runtime.wasm',
+	allowUnresolvedImports: true,
```

Comment:
added comment in: [2a95581](https://github.com/KILTprotocol/kilt-node/pull/614/commits/2a9558114f546cffa8078fa49cc473a97e42cd79)

Reply:
It explains what it does, but not why it is required. After reading the comment, I still don't understand if and when I need to use it.

Reply:
One more comment for this pls?

Reply:
The flag was needed, otherwise I receive an error, if I try to build the runtime from the WASM. 
[e2595f0](https://github.com/KILTprotocol/kilt-node/pull/614/commits/e2595f0b85e2da8a7a281cc3d7cd7c13aed07c28)